### PR TITLE
Add recipe for zbarlight

### PR DESCRIPTION
**Pull request by kdmurray91**
A python2 & py3 binding for zbar. Depends on #5971

### CODE REVIEW

**Comment by kdmurray91** [comment]
Hi @conda-forge/staged-recipes, this should be ready to review/merge now. Sorry for the delay in fixing the issues.

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes

File: recipes/zbarlight/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - zbar
+    - pillow
+  run:
+    - python
+    - pip
```
Comment (ocefpaf) [inline recipes/zbarlight/meta.yaml]: You probably don't need pip here.

File: recipes/zbarlight/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - setuptools
```
Comment (ocefpaf) [inline recipes/zbarlight/meta.yaml]: You'll need setuptools only if this package uses pkg_resources, can you confirm that?

File: recipes/zbarlight/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - zbar
+    - pillow
```
Comment (ocefpaf) [inline recipes/zbarlight/meta.yaml]: I'm assuming that zbar must be present at build time but I believe pillow is run-time only.

File: recipes/zbarlight/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [win]
```
Comment (ocefpaf) [inline recipes/zbarlight/meta.yaml]: Can you add a comment that you are skipping Windows b/c `zbar` is not available there?
If will help us answer questions in the future.

File: recipes/zbarlight/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (ocefpaf) [inline recipes/zbarlight/meta.yaml]: Please change to `script: "{{ PYTHON }} -m pip install . --no-deps -vv"`.

File: recipes/zbarlight/meta.yaml (hunk 3 above)
Comment (kdmurray91) [reply recipes/zbarlight/meta.yaml]: Pushed a change as you suggest, let's see if it builds

File: recipes/zbarlight/meta.yaml (hunk 1 above)
Comment (kdmurray91) [reply recipes/zbarlight/meta.yaml]: removed

File: recipes/zbarlight/meta.yaml (hunk 2 above)
Comment (kdmurray91) [reply recipes/zbarlight/meta.yaml]: it seems to, yes.

**Pull request merged**